README.md

```diff
@@ -22,6 +24,16 @@ cd terraform/
 tfenv install
 ```
 
+To use the code scans reusable workflow, add the following job to the jobs section of your CI workflow:
+
+```
+codeql-sast:
+    name: CodeQL SAST scan
+    uses: alphagov/govuk-infrastructure/.github/workflows/codeql-analysis.yml@main
+    permissions:
+        security-events: write
```

Comment:
For consistency with the rest of the YAML:

```suggestion
```yaml
codeql-sast:
  name: CodeQL SAST scan
  uses: alphagov/govuk-infrastructure/.github/workflows/codeql-analysis.yml@main
  permissions:
    security-events: write
```